frontend/packages/topology/src/filters/TopologyFilterBar.tsx

```diff
@@ -198,6 +199,15 @@ const TopologyFilterBar: React.FC<TopologyFilterBarProps> = ({
           variant={ToolbarGroupVariant['button-group']}
           alignment={{ default: 'alignRight' }}
         >
+          <ToolbarItem
+            className={
+              useFlag(ALLOW_EXPORT_APP) || kialiLink
```

Comment:
use hook in beginning of component defining and not directly here

Reply:
Updated